src/mapblock.cpp

```diff
@@ -559,16 +557,11 @@ void MapBlock::deSerialize(std::istream &is, u8 version, bool disk)
 void MapBlock::deSerializeNetworkSpecific(std::istream &is)
 {
 	try {
-		int version = readU8(is);
+		const u8 version = readU8(is);
```

Comment:
Should I keep this function for future usage or remove entirely?

Reply:
we will need it in the future, maybe, keep it